conf/core/ConfigOptions.asciidoc

```diff
@@ -1855,6 +1880,58 @@ This option allows for passing in a job ID for the currently executing command.
 trying to track multiple commands across a single job. When the Hootenanny API database is used
 for job output storage, the ID will be stored in the job status table of the associated map.
 
+=== josm.map.cleaner.add.detail.tags
+
+* Data Type: bool
+* Default Value: `false`
+
+If true, cleaning/validation detail tags will be added to map outputs when cleaning maps with JOSM.
+
+=== josm.map.validator.java.implementation
+
+* Data Type: string
+* Default Value: `hoot/core/josm/JosmMapValidator`
+
+File path relative to Hootenanny root directory without file extension or prefixed namespace to
+Java class to use as the JOSM Java map validator/cleaner implementation.
+
+=== josm.max.elements.for.map.string
+
+* Data Type: int
+* Default Value: `2000000`
+
+The maximum size in elements for a map allowed to be passed as a string between Hootenanny and JOSM.
+The default value was determined experimentally and may need tweaking per environment.
+
+=== josm.validators.exclude
+
+* Data Type: list
+* Default Value:
+** ``
+
+A list of JOSM validator Java class names without namespace prefixes or file extensions
+(e.g. 'DuplicatedWayNodes') to explicitly exclude use of during validation or cleaning using JOSM.
+Overrides values set in josm.validators.include.
+
+=== josm.validators.include
+
+* Data Type: list
+* Default Value:
+** ``
+
+A list of JOSM validator Java class names without namespace prefixes or file extensions
+(e.g. 'DuplicatedWayNodes') to explicitly include use of during validation or cleaning using JOSM.
+Overridden by values set in josm.validators.exclude.
+
+=== josm.validators.requiring.user.certificate
+
```

Comment:
I was told cert support is going away and not to run any of the validators that required it. This is a simple list to bypass the ones I have found to have a cert and we can remove this option after they no longer require a cert.